R/shape_by_id.R

```diff
@@ -0,0 +1,64 @@
+#' @title Link IDs to warterbody shapefiles
+#' 
+#' @param PERMANENT_match filter using PERMANENT_
+#' @param GNIS_ID_match filter using GNIS_ID
+#' @param GNIS_NAME_match filter using GNIS_NAME
+#' @param REACHCODE_match filer using REACHCODE
```

Comment:
Add a description indicating how these parameters are used (as in, data can be selected by different methods, but not all at the same time)